rest/config.go

```diff
@@ -767,9 +767,16 @@ func RunServer(config *ServerConfig) {
 
 	sc := NewServerContext(config)
 	for _, dbConfig := range config.Databases {
-		if _, err := sc.AddDatabaseFromConfig(dbConfig); err != nil {
+		c, err := sc.AddDatabaseFromConfig(dbConfig)
+		if err != nil {
 			base.LogFatal("Error opening database: %v", err)
 		}
+
```

Comment:
This wasn't in the ticket specification - apologies for that - but we only want to do this logging when c.UseGlobalSequence() == true.